taxonomies/ingredients.txt

```diff
@@ -11514,7 +11514,7 @@ wikipedia:en:https://en.wikipedia.org/wiki/Retinyl_acetate
 
 <en:vitamin A
 en:Retinyl Palmitate
-de:Retinylpalmitat
+de:Retinylpalmitat, Vitamin A
```

Comment:
```suggestion
de:Retinylpalmitat
```
Ah, the "Vitamin A" isn't needed, as the translation already exists in the parent entry. (The <en:... line indicates the entry's parent entry.) A bit further up in ingredients.txt is "de:Vitamin A". 

The structure here is that "vitamin A" is the generic term, and then there are child terms for specific forms of vitamin A, like beta carotene etc.

Another way to view the hierarchy (although it's not a live view of what's in github) is on the OFF site. E.g.
https://world.openfoodfacts.org/ingredient/vitamin-a
https://de.openfoodfacts.org/zutat/vitamin-a